components/locid/src/helpers.rs

```diff
@@ -120,6 +186,44 @@ impl<K, V> Store<K, V> for ShortVec<(K, V)> {
     }
 }
 
+#[inline]
+fn map_f_mut<K, V>(input: &mut (K, V)) -> (&K, &mut V) {
```

Comment:
nit: again, we can use closure syntax here, there's no need for a separate function that gets used just once

Reply:
Replaced with closure.